xsl/images/colorsvg/caution.svg

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Your whole xsl/images changes:
No, we do not want to replace the current OASIS images by our 20 years old versions!

Reply:
The check errors happen because there are still references to stylesheet.
Just click details in the failed checks.

Reply:
> Your whole xsl/images changes:
> No, we do not want to replace the current OASIS images by our 20 years old versions!

The resolution of current icons in stylesheets/ is too low for PDF files.
Do you have good SVG files or high-resolution files?

Reply:
https://github.com/gjanssens/gnucash/blob/maint/data/pixmaps/hicolor/scalable/apps/gnucash-icon.svg
is good.

Reply:
> The check errors happen because there are still references to stylesheet.
> Just click details in the failed checks.

I see.
I'm trying to change CMakeList.txt and CMakefiles/ right now.

Reply:
> The resolution of current icons in stylesheets/ is too low for PDF files.
> Do you have good SVG files or high-resolution files?

I'll change to use png for HTML, EPUB, CHM because 24x24 png is enough to display.
But I'd like to use SVG for PDF because 96x96 png is necessary.